jpa/boot-data-multipledatasources/src/main/java/com/example/multipledatasources/service/impl/DetailsServiceImpl.java

```diff
@@ -7,18 +7,22 @@
 import com.example.multipledatasources.repository.member.MemberRepository;
 import com.example.multipledatasources.service.DetailsService;
 import java.util.Optional;
-import lombok.RequiredArgsConstructor;
 import org.springframework.stereotype.Service;
 
 @Service
-@RequiredArgsConstructor
 public class DetailsServiceImpl implements DetailsService {
 
     private final CardHolderRepository cardHolderRepository;
     private final MemberRepository memberRepository;
 
+    public DetailsServiceImpl(CardHolderRepository cardHolderRepository, MemberRepository memberRepository) {
+        this.cardHolderRepository = cardHolderRepository;
+        this.memberRepository = memberRepository;
+    }
+
     @Override
     public ResponseDto getDetails(String memberId) {
+        // fetch data from database parallel
```

Comment:
@coderabbitai, can you please help me to fetch below two queries parallel to save time and then combine to produce resposne.